eos/utils/qcd_TEST.cc

```diff
@@ -38,24 +38,40 @@ class QCDMassesTest :
 
         virtual void run() const
         {
-            static const double eps = 1e-7;
-            static const std::vector<std::array<double, 5>> tests
+            // m_q_msbar(m_q_pole) at 3-loop
             {
-                                      // pole,  nf, alpha_s,   MSbar-ref
-                std::array<double, 5>{{ 172.0, 5.0,    0.10,  162.6620051 }},
-                std::array<double, 5>{{ 170.0, 5.0,    0.10,  160.7705865 }},
-                std::array<double, 5>{{ 168.0, 5.0,    0.10,  158.8791678 }},
-                std::array<double, 5>{{   4.9, 4.0,    0.22,    4.0271606 }},
-                std::array<double, 5>{{   4.8, 4.0,    0.22,    3.9449736 }},
-                std::array<double, 5>{{   4.7, 4.0,    0.22,    3.8627867 }},
-            };
-
-            for (const auto & test : tests)
-            {
-                double m_q_pole_input = test[0], nf = test[1], alpha_s = test[2], m_q_msbar_ref = test[3];
-                double m_q_msbar = QCD::m_q_msbar(m_q_pole_input, alpha_s, nf);
+                static const double eps = 1e-7;
+                static const std::vector<std::array<double, 5>> tests
+                {
+                                        // pole,  nf, alpha_s,   MSbar-ref
+                    std::array<double, 5>{{ 172.0, 5.0,    0.10,  162.6620051 }},
+                    std::array<double, 5>{{ 170.0, 5.0,    0.10,  160.7705865 }},
+                    std::array<double, 5>{{ 168.0, 5.0,    0.10,  158.8791678 }},
+                    std::array<double, 5>{{   4.9, 4.0,    0.22,    4.0271606 }},
+                    std::array<double, 5>{{   4.8, 4.0,    0.22,    3.9449736 }},
+                    std::array<double, 5>{{   4.7, 4.0,    0.22,    3.8627867 }},
+                };
+
+                for (const auto & test : tests)
+                {
+                    double m_q_pole_input = test[0], nf = test[1], alpha_s = test[2], m_q_msbar_ref = test[3];
+                    double m_q_msbar = QCD::m_q_msbar(m_q_pole_input, alpha_s, nf);
+
+                    TEST_CHECK_RELATIVE_ERROR(m_q_msbar, m_q_msbar_ref, eps);
+                }
+            }
 
-                TEST_CHECK_RELATIVE_ERROR(m_q_msbar, m_q_msbar_ref, eps);
+            // m_q_pole(m_q_msbar) at orders 1 to 3
+            {
+                static const double eps = 1e-12;
+                TEST_CHECK_NEARLY_EQUAL(QCD::m_q_pole(4.8, 0.22, 4.0, 0), 4.8,               eps);
```

Comment:
- ``n_f = 5.0`` vs ``n_f = 4.0``
 - MSbar scheme input value of 4.8 is quite high. Can you test for an input mass closer to m_b in the MSbar scheme at the scale ~5 GeV, which should be closer to 4 GeV.